robot/src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -120,6 +150,37 @@ private double addSlewRate(double joystickAxis){
         return slewRate.calculate(joystickAxis);
     }
 
+    public void resetOdometry(Pose2d pose) {
+        resetEncoders();
+        odometry.resetPosition(
+            gryo.getRotation2d(),
+            (frontLeftMotor.getEncoder().getPosition() / AutonomousConstants.ROTATIONS_PER_METER),
+            (frontRightMotor.getEncoder().getPosition() / AutonomousConstants.ROTATIONS_PER_METER),
```

Comment:
Here we use the individual motor encoder, and the value needs be correctly inverted

Setting the inverted on the motors would work here, setting it on the motorGroups would not.